src/libraries/ErrorsLib.sol

```diff
@@ -26,4 +26,10 @@ library ErrorsLib {
 
     /// @notice Thrown when rewards have already been claimed.
     string internal constant ALREADY_CLAIMED = "already claimed";
+
+    /// @notice Thrown when the value is already set.
+    string internal constant ALREADY_SET = "already set";
+
+    /// @notice Thrown when the submitted root (pending or not) is the same as the current one.
+    string internal constant ROOT_ALREADY_SET = "root already set";
```

Comment:
I would actually throw `ALREADY_SET` when the accepted root corresponds to the current root, and rename this error to `PENDING_ALREADY_SET` or something and throw it when the submitted root corresponds to the pending root (without necessarily stating that this error is specific to the root which is the only submittable value in the URD)